content/applications/sales/point_of_sale/payment/stripe.rst

```diff
@@ -0,0 +1,94 @@
+======
+Stripe
+======
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+Configuration
+=============
+
+Configure the payment method
+----------------------------
+
+Start by activating **Stripe** in the settings. From the **POS** dashboard, go to
+:menuselection:`Configuration --> Settings --> Payment Terminals --> Check Stripe`.
+
+Then, create the payment method:
+
+- Click :menuselection:`Configuration --> Payment Methods --> Create`, and complete the
+  :guilabel:`Method` field with your payment method's name;
+- Set the :guilabel:`Journal` field as :guilabel:`Bank` and the :guilabel:`Use a Payment Terminal`
+  field as :guilabel:`Stripe`;
+- Enter your payment terminal serial number in the :guilabel:`Stripe Serial Number` field;
+- Click on :guilabel:`Don't forget to complete Stripe connect before using this payment method.`
+
+.. image:: stripe/create-method-stripe.png
+
+.. note::
+   - Find your payment terminal serial number under the device or on `Stripe's website <https://dashboard.stripe.com/register>`_.
+
+Link Stripe with Odoo
+---------------------
+
+Click on :guilabel:`Connect Stripe`. Doing so redirects you automatically to a configuration page.
+Fill in all the information to create your Stripe account and link it with Odoo. Once the forms are
+completed, the :guilabel:`Publishable Key` and the :guilabel:`Secret Key` are automatically loaded
+in Odoo, and your terminal is ready to be used in a POS.
+
+.. image:: stripe/stripe-connect.png
+
+.. important::
+   - When you use **Stripe** exclusively in PoS, you only need the **Secret Key** to use your
+     terminal.
+   - The :guilabel:`State` is inherent to **eCommerce** and can stay set as :guilabel:`Disabled`.
+   - For **On-Premise** users, the :guilabel:`Connect Stripe` button does not work. To retrieve the
+     keys manually, go to `Stripe's website <https://dashboard.stripe.com/register>`_, log in, type
+     `API` in the search bar, and click :guilabel:`Developers > API`.
+
+Configure the payment terminal
+------------------------------
+
+Swipe right on your payment terminal, click :menuselection:`Settings --> enter the admin PIN code
+--> Validate --> Network`, and select your network.
+
+.. note::
+   - The device must be connected to a secured WIFI.
+   - Your **Odoo** database and payment terminal must share the same network.
+   - To access your payment terminal settings, you must enter the admin PIN code. By default, this
+     code is `07139`.
+
+Add a new payment method
+------------------------
+
+To add a new **payment method** to your point of sale, go to :menuselection:`Configuration -->
+Settings`. Select the POS, go to :menuselection:`Payments --> Payment Methods`, and add your payment
+method for **Stripe**.
```

Comment:
Hi @LoredanaLrpz, nice one, I like it !

Just one suggestion here, "Add a new payment method" sounds redundant with the previous "Configure a payment method" IMHO, although those sections speaks well about different things.

To avoid confusion, why not:
* renaming this section "Link the new payment method"
* and start with "To add the newly configured **payment method** to your point of sale, ...."

What do you think ? Sorry if I sound peaky :)